src/main/kotlin/com/openlattice/linking/MatchedEntityPair.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2019. OpenLattice, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * You can contact the owner of the copyright at support@openlattice.com
+ *
+ *
+ */
+
+package com.openlattice.linking
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.openlattice.client.serialization.SerializationConstants
+
+data class MatchedEntityPair(
+        @JsonProperty(SerializationConstants.ENTITY_KEY_IDS) val entityPair: EntityKeyPair,
+        @JsonProperty(SerializationConstants.MATCH) val match: Double) {
+
+    override fun equals(other: Any?): Boolean {
```

Comment:
Yes, so it will only take the entityPair field for comparison and ignore the match value. (we cannot have different values for same entity pair)
I will fix the hashcode, to reflect that.. don't know why I left it like that